src/cypress/utils/navigation.ts

```diff
@@ -8,29 +8,80 @@ import { resolveIdentifier } from './index'
 interface NavigationOptions {
   categoryId: string
   random: boolean
+  pageIndex?: number
 }
 
-export function goToSearchPage(options?: NavigationOptions) {
-  if (!options) return
+export function goToSearchPage(options?: NavigationOptions): boolean {
```

Comment:
This method became a little bit huge, can you extract it or, if you think that can't be decoupled, can you comment on each major step?

Reply:
It was unnecessarily huge, refactored it in the latest version